src/gt4py/next/program_processors/runners/dace_iterator/__init__.py

```diff
@@ -70,21 +70,32 @@ def get_shape_args(
 def get_stride_args(
     arrays: Mapping[str, dace.data.Array], args: Mapping[str, Any]
 ) -> dict[str, Any]:
-    return {
-        str(sym): size / value.itemsize
-        for name, value in args.items()
-        for sym, size in zip(arrays[name].strides, value.strides)
-    }
+    stride_args = {}
```

Comment:
I guess that the type annotation can be derived directly from the return type of the function, in this case. Adding the type annotation on this variable seems a bit redundant to me. One thing I react about, now when I look more carefully at type annotations in this function, is that the type of the return value should be a bit more specific to make it useful. I am going to change it to `dict[str, int]` (and same for shape) since shape and stride are integer values.

Reply:
I guess that the type annotation can be derived directly from the return type of the function, in this case. Adding the type annotation on this variable seems a bit redundant to me.
One thing I react about, now when I look more carefully at type annotations in this function, is that the type of the return value should be a bit more specific to make it useful. I am going to change it to dict[str, int] (and same for shape) since shape and stride are integer values.

Reply:
Yeah, that's what I was guessing as well. In that case it's good like this.